operations/services.md

```diff
@@ -395,6 +398,7 @@ Returns all rates \(pricing setups\) and rate groups \(condition settings\) of t
 | `IsPublic` | boolean | required | Whether the rate is publicly available. |
 | `Name` | string | required | Name of the rate. |
 | `ShortName` | string | required | Short name of the rate. |
+| `ExternalNames` | array | required | External names of the rate for each language-culture code they were     translated to. |
```

Comment:
It is same as here: https://mews-systems.gitbook.io/connector-api/operations/enterprises#localized-text

![image](https://user-images.githubusercontent.com/435787/83693876-3d2bb080-a5f7-11ea-9905-d421174ce0c2.png)

Reply:
Thank you